lms/djangoapps/commerce/views.py

```diff
@@ -140,6 +141,14 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         except (exceptions.SlumberBaseException, exceptions.Timeout) as ex:
             log.exception(ex.message)
             return InternalRequestErrorResponse(ex.message)
+        finally:
+            audit_log(
```

Comment:
Since Otto's basket endpoint places and fulfills orders immediately after checkout for free baskets, this "checkout_requested" message will be logged *after* messages corresponding to a free checkout: "basket_frozen," "order_placed," "enrollment_change_requested," and "line_fulfilled." Thoughts on this? Since our monitoring queries will be looking for summary statistics and won't be sensitive to the exact order in which these messages are emitted, I think this is okay.